import-service/test/handlers/import-products-file.test.js

```diff
@@ -0,0 +1,72 @@
+jest.mock('../../src/services/s3-client');
+
+import * as response from '../../src/services/response';
+import * as s3 from '../../src/services/s3-client';
+import { importProductsFile } from '../../src/handlers/import-products-file';
+
+import { event } from '../mocks/created-file-event.mock';
+
+beforeEach(() => {
+  response.ok = jest.fn();
+  response.badRequest = jest.fn();
+  response.serverError = jest.fn();
+  s3.getPutSignedUrlPromise = jest.fn();
+  global.console = {
+    info: jest.fn(),
+    error: jest.fn(),
+  };
+});
+
+afterEach(() => {
```

Comment:
btw, do u really need this block? u recreate mock every time in before each block. 
at least u could use `jest.clearAllMocks()`  https://jestjs.io/ru/docs/jest-object#jestclearallmocks instead of the 21-26 lines